Testing-Web-Applications-in-Spring-Boot/src/test/java/testing/Web/Applications/in/Spring/Boot/controller/StudentControllerTest.java

```diff
@@ -0,0 +1,99 @@
+package testing.Web.Applications.in.Spring.Boot.controller;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.bean.override.mockito.MockitoBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import testing.Web.Applications.in.Spring.Boot.exception.StudentNotFoundException;
+import testing.Web.Applications.in.Spring.Boot.model.Faculty;
+import testing.Web.Applications.in.Spring.Boot.model.Student;
+import testing.Web.Applications.in.Spring.Boot.service.StudentService;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(testing.Web.Applications.in.Spring.Boot.controller.StudentControllerTest.class)
```

Comment:
```suggestion
@WebMvcTest(StudentController.class)
```
Указываем контроллер, который нужно протестировать